js/index.js

```diff
@@ -65,3 +65,47 @@ messageForm.addEventListener('submit', (e) => {
 
     messageForm.reset();
 });
+
+let githubRequest = new XMLHttpRequest();
+
+githubRequest.open("GET","https://api.github.com/users/up2code3/repos")
+githubRequest.send();
+
+addEventListener("load", (Event) => {
+  let repositories = JSON.parse(githubRequest.responseText);
+  console.log(repositories)
+
+
+// I compared the next to lines of code to the message section above to make sure it was right 
+let projectSection = document.getElementById("projects") 
+let projectList = projectSection.querySelector("ul") // this queries just the project section and not the whole page  
+
+
+for (var i = 0 ; i < repositories.length ; i++){
+  let project = document.createElement("li");  
+  project.innerText = repositories[i].name; 
+  projectList.appendChild(project);
+}
+})
+
+//  I was stuck on line 87 for awhile, it had 7 li under the 
```

Comment:
You can do multi-line comments like so:

```
/*
This
is
a multi-line
comment
*/
```

`// this is a single line comment`

Reply:
Also, a comment that asks questions or seeks to explain something to a reviewer should be written as a comment in the PR (like I am doing with this comment). This way we can have a conversation about your issues, and not have unnecessary comments in source code.